README.md

```diff
@@ -208,7 +208,7 @@ docker build --build-arg arch=arm64 .
 
 This uses Docker and QEMU to achieve an ARM64 build on CircleCI's x86 hardware.
 
- * **TODO!** [Build]()
+ * [Build](https://circleci.com/gh/htm-community/nupic.cpp/tree/master)
```

Comment:
I think atleast this change has been fixed in a recent PR, so I'll close this PR.